pkg/asset/manifests/template.go

```diff
@@ -54,6 +54,14 @@ type OvirtCredsSecretData struct {
 	Base64encodeCABundle string
 }
 
+// NutanixCredsSecretData holds encoded credentials and is used to generate cloud-creds secret
+type NutanixCredsSecretData struct {
+	Base64encodeNutanixEndpoint string
+	Base64encodeNutanixPort     string
+	Base64encodeNutanixUsername string
+	Base64encodeNutanixPassword string
```

Comment:
"Nutanix" in these names is redundant.
```suggestion
	Base64encodeEndpoint string
	Base64encodePort     string
	Base64encodeUsername string
	Base64encodePassword string
```